src/Microsoft.Tye.Core/ConfigModel/NameInferer.cs

```diff
@@ -21,7 +21,7 @@ internal static class NameInferer
                 return Path.GetFileNameWithoutExtension(fileInfo.Name).ToLowerInvariant();
             }
 
-            return fileInfo.Directory?.Parent?.Name.ToLowerInvariant();
+            return fileInfo.Directory?.Name.ToLowerInvariant();
```

Comment:
Why did this change?

Reply:
This was incorrect, it should be the current directory name for the name of the app, rather than the parent.